pkg/trace/stats/statsraw.go

```diff
@@ -139,11 +139,11 @@ func (sb *RawBucket) Export() Bucket {
 		}
 	}
 	for k, v := range sb.sublayerData {
-		key := GrainKey(k.name, k.measure, k.aggr)
+		key := GrainKey(k.name, k.sub.Metric, k.aggr+","+k.sub.Tag.Name+":"+k.sub.Tag.Value)
```

Comment:
Have you considered or tried reusing the same `strings.Builder` in each iteration to concatenate the strings? There could be a benefit.

Reply:
I've thought about it, but the frequency of this operation becomes rather insignificant once it's done on flushes rather than on inserts (it's not even visible on the CPU profiles anymore) that I thought it would not bring much additional benefit and slightly complexify the code. But I can look into it, it's a small change.

Reply:
I'll leave it up to you to decide... Fine by me.

Reply:
I'll leave it like that for this PR and may revisit later if we see it in profiles.